lib/src/view/game/game_list_tile.dart

```diff
@@ -402,3 +409,101 @@ class _ContextMenu extends ConsumerWidget {
     );
   }
 }
+
+class ExtendedGameListTile extends StatelessWidget {
```

Comment:
We don't really understand what this widget does just by reading its name. Since it's a public class it should have a doc comment explaining what it is.
And if possible use a better name.

Reply:
What does it add to the existing `GameTile` and why is it needed? not sure to understand.